SA3/qa_web/templates/qa_web/question_display_page.html

```diff
@@ -70,51 +70,54 @@ <h3><a href="{% url 'answers' question.id %}">{{ question.title }}</a></h3>
                     {% if question.num_answers > 3 %}
                     <span style="padding:0.85%" class="fas fa-fire" aria-hidden="true"></span>{% endif %}
                     <br/>
-{#                    {% for tag in question.tags.all %}#}
-{#                        <a href="{% url 'some_tag_link' tag %}"><tag>{{ tag.name }}</tag></a>  {# to-do: add tag link #}
-{#                    {% endfor %}#}
+
+
                     <div class="container" align="right">
-                    <small> {{ question.creation_date }}</small>
-                        </div>
+                        {% for tag in question.tag.all %}
+                          {% if tag.slug != '' %}
+                            tag:<a href="{% url 'question_by_tag' tag.slug %}"><tag>{{ tag.slug }}</tag></a>
```

Comment:
Here it would be nice to not see multiple 'tag' labels if there are many tags. one 'tag' label is enough.

![image](https://user-images.githubusercontent.com/29107916/36947518-033ba8e2-1f9b-11e8-8313-1faa066e4f0a.png)

Reply:
Good point. I will make it once.